Lib/test/test_sys.py

```diff
@@ -1222,7 +1222,7 @@ def get_gen(): yield 1
         # list
         samples = [[], [1,2,3], ['1', '2', '3']]
         for sample in samples:
-            check(sample, vsize('Pn') + len(sample)*self.P)
+            check(list(sample), vsize('Pn') + len(sample)*self.P)
```

Comment:
This is a `sys.getsizeof` test. The 3-element lists are overallocated, so we resize them here.